gecoscc/views/settings.py

```diff
@@ -28,6 +28,9 @@
 from bson.objectid import ObjectId
 logger = logging.getLogger(__name__)
 
+
```

Comment:
What is this for?

Reply:
This settings, which are not set in the view, are saved in MongoDB collection.